.gitignore

```diff
@@ -52,3 +52,11 @@ yarn-debug.log*
 .env.test.local
 .env.production.local
 .env.local
+
+
+.gradle/
+Gemfile.lock
```

Comment:
You remove Gemfile.lock but since it's already been checked-in, it won't retroactively ignore it. Need to either delete the file (this is fine, because the docker image sets it up anyway) or reset the git cache so the .gitignore gets adhered to.

Also, I think to avoid this in the future, we should probably create/update a .dockerignore file that has `node_modules`, `.git`, and `Gemfile.lock`